bin/wp-cli.php

```diff
@@ -82,6 +82,31 @@ public function flush() {
 		}
 	}
 
+	/**
+	 * Add date range when retrieving posts in bulk.
+	 * Dates need to be passed as mmddyyy. See synopsis for index function.
+	 *
+	 * @param $args array
+	 * @return $args array
+	 */
+	public function apply_date_range( $args ) {
```

Comment:
WP-CLI may see this as a subcommand because it's public. Can you verify that it doesn't (`wp help searchpress`)? If it does, I'm sure there's a phpdoc flag to indicate that it isn't.